overlord/ifacestate/ifacemgr.go

```diff
@@ -176,6 +177,10 @@ func (m *InterfaceManager) StartUp() error {
 
 	ifacerepo.Replace(s, m.repo)
 
+	// XXX: replaces current callback with one from the current ifacemgr
```

Comment:
this is a pattern we use elsewhere,  we want a comment like:

// wire late profile removal support into snapstate